packages/connect-ui/src/index.tsx

```diff
@@ -72,8 +72,14 @@ export const ConnectUI = ({ postMessage, clearLegacyView }: ConnectUIProps) => {
         } else {
             // messages[0] could be null. in that case, legacy view is rendered
             switch (messages[0]?.type) {
+                case 'waiting-for-iframe-init':
+                    component = <Loader message="waiting for iframe to load" />;
+                    break;
+                case 'waiting-for-iframe-handshake':
+                    component = <Loader message="waiting for handshake from iframe" />;
```

Comment:
Nit: messages seam too technical for End user facing UI. Probably final user does not need to know what `iframe` is.

Reply:
true, I'll change it to something more readable.